src/wallet-watcher.ts

```diff
@@ -204,7 +208,12 @@ const checkAndFundWallet = async (
       walletConfig.opsGenieConfig
     );
 
-    const gasOptions = await getGas(globalSponsor);
+    const [logs, gasTarget] = await getGasPrice(wallet.provider, walletConfig.chains[chainId].options);
+    logs.forEach((log) =>
+      logging.log(log.message, log.level === 'INFO' || log.level === 'ERROR' ? log.level : undefined)
+    );
+
+    const { gasLimit: _gasLimit, ...restGasTarget } = gasTarget;
```

Comment:
Currently removing the gasLimit but this can be added back if we'd want to allow customizing it. It is required in the ChainOptions type so this would need to be set to a real value if used.